src/components/DocumentProducer/DeliverableVariableDetailsInput/index.tsx

```diff
@@ -307,14 +309,32 @@ const DeliverableVariableDetailsInput = ({
         />
       )}
 
-      {variable.type === 'Select' && (
+      {variable.type === 'Select' && !variable.isMultiple && (
         <Dropdown
-          onChange={(newValue: any) => onChangeValueHandler(newValue, 'value')}
+          fullWidth
           label=''
+          onChange={(newValue: any) => onChangeValueHandler(newValue, 'value')}
           options={getOptions()}
-          selectedValue={value}
-          fullWidth={true}
+          selectedValue={(value as number[])?.[0]}
+          sx={[formElementStyles, { paddingBottom: theme.spacing(1) }]}
```

Comment:
Most of these lines are just re-ordering props, but I also removed the redundant `={true}` from `fullWidth` and changed the value of `selectedValue` from `value` to `(value as number[])?.[0]`.